src/hook.js

```diff
@@ -1,7 +1,7 @@
 import { useCallback, useState, useRef } from 'react'
 import * as Utils from './utils'
 
-export default function useUncontrolled(props, config) {
+export function useUncontrolled(props, config) {
```

Comment:
is this a preference for only named exports or does it affect something meaningful as an internal package file?

Reply:
Nothing is affected. I just personally not use default exports at all. Do you want to revert this?

Reply:
yeah i think so, in general we/me prefer single exports being defaults. its more consistent with linting i've got to deal with professionally